src/backend/marsha/deposit/migrations/0002_alter_filedepository_description_and_more.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 4.0.7 on 2022-09-20 07:30
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
NIT: It seems this migration does not apply any change on database, I would have add a "noop" prefix to the name.

Reply:
Done